src/styles/blocks/_watch.scss

```diff
@@ -0,0 +1,41 @@
+.wrapper {
```

Comment:
Move wrapper to it's own file, each BEM block should be in separate file, especially as there is no need to nest .watch class inside in this case